dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -822,7 +823,8 @@ public void findByObjectAndFeatureTest() throws Exception {
         String epersonToken = getAuthToken(eperson.getEmail(), password);
         getClient(epersonToken).perform(get("/api/authz/authorizations/search/objectAndFeature")
                 .param("uri", comUri)
-                .param("projection", "full")
+                .param("projection", "level")
```

Comment:
It could, but regardless we want to remove the full projection as much as possible as angular shouldn't be using this neither.

Reply:
please keep it in place if there is no immediate needs to remove it.
It would be valuable add an extra call with the level projection to cover also the normal use case from angular